js/reference.js

```diff
@@ -836,18 +836,47 @@
                 // 3) not all visible columns in the table are generated
                 var ref = (this._context === module._contexts.CREATE) ? this : this.contextualize.entryCreate;
 
-                this._canCreate = ref._table.kind !== module._tableKinds.VIEW && !ref._table._isGenerated && ref._checkPermissions("insert");
+                if (ref._table.kind === module._tableKinds.VIEW) {
+                    this._canCreate = false;
+                    this._canCreateReason = "Table is a view. ";
```

Comment:
Can you please add an object like the ones that we have [at the bottom of utilities.js](https://github.com/informatics-isi-edu/ermrestjs/blob/master/js/utilities.js#L3182) to have these messages in one place?